tools/yoink/templates/README.md.j2

```diff
@@ -0,0 +1,11 @@
+# `source-{{source_name}}`
```

Comment:
💅 I would recommend we hoist `source-` into a `connector_image_name` property both for reuse and future destination support